### DISCUSSION TITLE
R 5.1.3.0 Development Phase Sunbird Cokreat Building Block QA Validation Status

### DISCUSSION THREAD

**Discussion by Nadiya496** [Sunbird-coKreat/Community#49] [General Discussions]
Hello All,

Please find the Building Block QA Validation Status of  "Sunbird coKreat" for R5.1.3.0 

Environment: Dock Staging
Testing Status : **Not yet started**
Planned Testing Closure: 31 Jan 2023
Defects Dashboard: NA
Execution Sheet: [Link](https://docs.google.com/spreadsheets/d/1TTu1XeL8XJqKCjNVN501rx9UJJSZwbB4xNamJD0ls2g/edit?usp=sharing)

_**Scope:**_ 
Release 5.1.1.0 : [Link](https://project-sunbird.atlassian.net/issues/?filter=12693)
Release 5.1.2.0 : [Link](https://project-sunbird.atlassian.net/issues/?filter=12716)
Release 5.1.3.0 : [Link](https://project-sunbird.atlassian.net/issues/?filter=12736)

**NOTE: As discussed with @luckynvdu @@mmanimegalai @ @chitranshu-keshav All the items of R 5.1.1.0 and R5.1.2.0 will be merged along with R 5.1.3.0 will be delivered for QA validation. But currently we are unable to start the validation of those tickets since release branch 5.1.3.0 is not yet created by the Sunbird ED building block.**

Test Case Review : Completed. (Relevant scenarios are been reviewed by the PO)
Test Scenarios: [Confluence Link](https://project-sunbird.atlassian.net/wiki/spaces/COK/pages/3272114311/R+5.1.1.0+Test+Scenarios)

cc: @vinukumar-vs  @chitranshu-keshav @ShailandraOjha   @Aparna-karri591 @Nadiya496  @ashokreddy1208 @mmanimegalai @arun1205 @vaivk369

**Comment by mmanimegalai** [comment]
@Krishnaj20 @vinukumar-vs  Please let us know if there an update on this
cc @ShailandraOjha @Aparna-karri591 @Nadiya496

**Comment by Aparna-karri591** [comment]
Hello All,

Please find the Building Block QA Validation Status of "Sunbird coKreat" for R5.1.3.0

Environment: Dock Staging
Testing Status : Not yet started
Planned Testing Closure: 31 Jan 2023
Defects Dashboard: NA
Execution Sheet: [Link](https://docs.google.com/spreadsheets/d/1TTu1XeL8XJqKCjNVN501rx9UJJSZwbB4xNamJD0ls2g/edit?usp=sharing)

Scope:
Release 5.1.1.0 : [Link](https://project-sunbird.atlassian.net/issues/?filter=12693)
Release 5.1.2.0 : [Link](https://project-sunbird.atlassian.net/issues/?filter=12716)
Release 5.1.3.0 : [Link](https://project-sunbird.atlassian.net/issues/?filter=12736)


 **NOTE: As discussed with @luckynvdu @@mmanimegalai @ @chitranshu-keshav All the items of R 5.1.1.0 and R5.1.2.0 will be merged along with R 5.1.3.0 will be delivered for QA validation. But currently we are unable to start the validation of those tickets since release branch 5.1.3.0 is not yet created by the Sunbird ED building block.**

**Test Case Review** : Completed for all tickets (except https://project-sunbird.atlassian.net/browse/CO-267 as per today's discussion with @vinukumar-vs we have to write scenarios for  program-service & Creation-portal backend)   . 
Test Scenarios: [Confluence Link](https://project-sunbird.atlassian.net/wiki/spaces/COK/pages/3272114311/R+5.1.1.0+Test+Scenarios)

cc: @vinukumar-vs @chitranshu-keshav @ShailandraOjha @Aparna-karri591 @Nadiya496 @ashokreddy1208 @mmanimegalai @arun1205 @vaivk369

**Comment by Aparna-karri591** [comment]
Hello All,

Please find the Building Block QA Validation Status of "Sunbird coKreat" for R5.1.3.0

Environment: Dock Staging
Testing Status : Not yet started
Planned Testing Closure: 31 Jan 2023
Defects Dashboard: NA
Execution Sheet: [Link](https://docs.google.com/spreadsheets/d/1TTu1XeL8XJqKCjNVN501rx9UJJSZwbB4xNamJD0ls2g/edit?usp=sharing)

Scope:
Release 5.1.1.0 : [Link](https://project-sunbird.atlassian.net/issues/?filter=12693)
Release 5.1.2.0 : [Link](https://project-sunbird.atlassian.net/issues/?filter=12716)
Release 5.1.3.0 : [Link](https://project-sunbird.atlassian.net/issues/?filter=12736)

NOTE: As discussed with @luckynvdu @@mmanimegalai @ @chitranshu-keshav All the items of R 5.1.1.0 and R5.1.2.0 will be merged along with R 5.1.3.0 will be delivered for QA validation. But currently we are unable to start the validation of those tickets since release branch 5.1.3.0 is not yet created by the Sunbird ED building block.

Test Case Review : Completed for all tickets (except https://project-sunbird.atlassian.net/browse/CO-267 as per today's discussion with @vinukumar-vs we have to write scenarios for program-service & Creation-portal backend) .
Test Scenarios: [Confluence Link](https://project-sunbird.atlassian.net/wiki/spaces/COK/pages/3272114311/R+5.1.1.0+Test+Scenarios)

cc: @vinukumar-vs @chitranshu-keshav @ShailandraOjha @Aparna-karri591 @Nadiya496 @ashokreddy1208 @mmanimegalai @arun1205 @vaivk369

**Comment by Aparna-karri591** [comment]
Hello All,

Please find the Building Block QA Validation Status of "Sunbird coKreat" for R5.1.3.0

Environment: Dock Staging
Testing Status : Not yet started
Planned Testing Closure: 31 Jan 2023
Defects Dashboard: NA
Execution Sheet: [Link](https://docs.google.com/spreadsheets/d/1TTu1XeL8XJqKCjNVN501rx9UJJSZwbB4xNamJD0ls2g/edit?usp=sharing)

**Scope:**
Release 5.1.1.0 : [Link](https://project-sunbird.atlassian.net/issues/?filter=12693)
Release 5.1.2.0 : [Link](https://project-sunbird.atlassian.net/issues/?filter=12716)
Release 5.1.3.0 : [Link](https://project-sunbird.atlassian.net/issues/?filter=12736)


 **NOTE:** 

1. As discussed with @luckynvdu @@mmanimegalai @ @chitranshu-keshav All the items of R 5.1.1.0 and R5.1.2.0 will be merged along with R 5.1.3.0 will be delivered for QA validation. .
2.  Testing is not yet started as still build number is showing 5.1.0

Test Case Review : Completed for all tickets

Test Scenarios: [Confluence Link](https://project-sunbird.atlassian.net/wiki/spaces/COK/pages/3272114311/R+5.1.1.0+Test+Scenarios)

cc: @vinukumar-vs @chitranshu-keshav @ShailandraOjha @Aparna-karri591 @Nadiya496 @ashokreddy1208 @mmanimegalai @arun1205 @vaivk369  @Krishnaj20 @rajeevsathish

**Comment by Aparna-karri591** [comment]
Hello All,

Please find the Building Block QA Validation Status of "Sunbird coKreat" for R5.1.3.0

**Environment:** Dock Staging
**Testing Status :** 
![image](https://user-images.githubusercontent.com/106593060/217269025-16fbf678-db60-4384-8ad9-68e6affcf6cf.png)

**Planned Testing Closure**: 31 Jan 2023
**Defects** : CO-304 and CO-305
**Execution Sheet**: [Link](https://docs.google.com/spreadsheets/d/1TTu1XeL8XJqKCjNVN501rx9UJJSZwbB4xNamJD0ls2g/edit?usp=sharing)
**Scope:**
Release 5.1.1.0 : [Link](https://project-sunbird.atlassian.net/issues/?filter=12693)
Release 5.1.2.0 : [Link](https://project-sunbird.atlassian.net/issues/?filter=12716)
Release 5.1.3.0 : [Link](https://project-sunbird.atlassian.net/issues/?filter=12736)

NOTE:
As discussed with @luckynvdu @@mmanimegalai @ @chitranshu-keshav All the items of R 5.1.1.0 and R5.1.2.0 will be merged along with R 5.1.3.0 will be delivered for QA validation. .
Test Case Review : Completed for all tickets
Test Scenarios: [Confluence Link](https://project-sunbird.atlassian.net/wiki/spaces/COK/pages/3272114311/R+5.1.1.0+Test+Scenarios)

cc: @vinukumar-vs @chitranshu-keshav @ShailandraOjha @Aparna-karri591 @Nadiya496 @ashokreddy1208 @mmanimegalai @arun1205 @vaivk369 @Krishnaj20 @rajeevsathish

**Comment by Aparna-karri591** [comment]
Hello All,

Please find the Building Block QA Validation Status of "Sunbird coKreat" for R5.1.3.0

Environment: Dock Staging

Testing Status : 

![image](https://user-images.githubusercontent.com/106593060/217546695-aa4eaa11-acf4-4557-9d4c-1b48b28c4997.png)


Planned Testing Closure: 31 Jan 2023
Defects : [CO-304](https://project-sunbird.atlassian.net/browse/CO-304) and [CO-305](https://project-sunbird.atlassian.net/browse/CO-305)
Execution Sheet: [Link](https://docs.google.com/spreadsheets/d/1TTu1XeL8XJqKCjNVN501rx9UJJSZwbB4xNamJD0ls2g/edit?usp=sharing)

Scope:
Release 5.1.1.0 : [Link](https://project-sunbird.atlassian.net/issues/?filter=12693)
Release 5.1.2.0 : [Link](https://project-sunbird.atlassian.net/issues/?filter=12716)
Release 5.1.3.0 : [Link](https://project-sunbird.atlassian.net/issues/?filter=12736)

NOTE:
As discussed with @luckynvdu @@mmanimegalai @ @chitranshu-keshav All the items of R 5.1.1.0 and R5.1.2.0 will be merged along with R 5.1.3.0 will be delivered for QA validation. .
Test Case Review : Completed for all tickets
Test Scenarios: [Confluence Link](https://project-sunbird.atlassian.net/wiki/spaces/COK/pages/3272114311/R+5.1.1.0+Test+Scenarios)

cc: @vinukumar-vs @chitranshu-keshav @ShailandraOjha @Aparna-karri591 @Nadiya496 @ashokreddy1208 @mmanimegalai @arun1205 @vaivk369 @Krishnaj20 @rajeevsathish

**Comment by Aparna-karri591** [comment]
Hello ,

Currently we are not able to provide the Sign Off since [CO-172](https://project-sunbird.atlassian.net/browse/CO-172) QA Required Ticket is not yet released and we have 2 Open defects([CO-304](https://project-sunbird.atlassian.net/browse/CO-304) and [CO-305](https://project-sunbird.atlassian.net/browse/CO-305))

cc: @vinukumar-vs @chitranshu-keshav @ShailandraOjha @Nadiya496 @ashokreddy1208 @mmanimegalai @arun1205 @vaivk369 @Krishnaj20 @rajeevsathish

**Comment by Aparna-karri591** [comment]
Hello ,

Currently we are not able to provide the Sign Off since [CO-172](https://project-sunbird.atlassian.net/browse/CO-172) (QA Required)
 Ticket is not yet released and we have 2 Open defects([CO-304](https://project-sunbird.atlassian.net/browse/CO-304) and [CO-305](https://project-sunbird.atlassian.net/browse/CO-305))

**Note**:  Two developers are on leave

cc: @vinukumar-vs @chitranshu-keshav @ShailandraOjha @Nadiya496 @ashokreddy1208 @mmanimegalai @arun1205 @vaivk369 @Krishnaj20 @rajeevsathish

**Comment by Aparna-karri591** [comment]
Hello ,

Currently we are not able to provide the Sign Off since [CO-172](https://project-sunbird.atlassian.net/browse/CO-172) (QA Required)
Ticket is not yet released and we have 2 Open defects([CO-304](https://project-sunbird.atlassian.net/browse/CO-304) and [CO-305](https://project-sunbird.atlassian.net/browse/CO-305))

**Note: Two developers are on leave**

cc: @vinukumar-vs @chitranshu-keshav @ShailandraOjha @Nadiya496 @ashokreddy1208 @mmanimegalai @arun1205 @vaivk369 @Krishnaj20 @rajeevsathish

**Comment by Aparna-karri591** [comment]
Hello ,

Currently we are not able to provide the Sign Off since [CO-172](https://project-sunbird.atlassian.net/browse/CO-172) (QA Required)
Ticket is not yet released and we have 2 Open defects([CO-304](https://project-sunbird.atlassian.net/browse/CO-304) and [CO-305](https://project-sunbird.atlassian.net/browse/CO-305))

**Note: Two developers are on leave**

cc: @vinukumar-vs @chitranshu-keshav @ShailandraOjha @Nadiya496 @ashokreddy1208 @mmanimegalai @arun1205 @vaivk369 @Krishnaj20 @rajeevsathish

**Comment by Aparna-karri591** [comment]
Hello ,

Currently we are not able to provide the Sign Off since publish job is failing on dock staging,  [CO-172](https://project-sunbird.atlassian.net/browse/CO-172) (QA Required)
Ticket is not yet released and we have 2 Open defects([CO-304](https://project-sunbird.atlassian.net/browse/CO-304) and [CO-305](https://project-sunbird.atlassian.net/browse/CO-305))

Note: one developer is on leave

cc: @vinukumar-vs @chitranshu-keshav @ShailandraOjha @Nadiya496 @ashokreddy1208 @mmanimegalai @arun1205 @vaivk369 @Krishnaj20 @rajeevsathish

**Comment by Aparna-karri591** [comment]
Hello ,

Currently we are not able to provide the Sign Off since publish API  is failing on dock staging, [CO-172](https://project-sunbird.atlassian.net/browse/CO-172) (QA Required)
Ticket is not yet released and we have 2 Open defects([CO-304](https://project-sunbird.atlassian.net/browse/CO-304) and [CO-305](https://project-sunbird.atlassian.net/browse/CO-305))

**Note: one developer is on leave**

cc: @vinukumar-vs @chitranshu-keshav @ShailandraOjha @Nadiya496 @ashokreddy1208 @mmanimegalai @arun1205 @vaivk369 @Krishnaj20 @rajeevsathish

**Comment by Aparna-karri591** [comment]
Hello ,

Currently we are not able to provide the Sign Off since publish API is failing on dock staging, [CO-172](https://project-sunbird.atlassian.net/browse/CO-172) (QA Required)
Ticket is not yet released and we have 1 Open defect [CO-305](https://project-sunbird.atlassian.net/browse/CO-305)

Note: one developer is on leave

cc: @vinukumar-vs @chitranshu-keshav @ShailandraOjha @Nadiya496 @ashokreddy1208 @mmanimegalai @arun1205 @vaivk369 @Krishnaj20 @rajeevsathish

**Comment by Aparna-karri591** [comment]
Hello ,

Currently we are not able to provide the Sign Off since [CO-172](https://project-sunbird.atlassian.net/browse/CO-172) QA Required ticket is not yet released 
Note: one developer is on leave

cc: @vinukumar-vs @chitranshu-keshav @ShailandraOjha @Nadiya496 @ashokreddy1208 @mmanimegalai @arun1205 @vaivk369 @Krishnaj20 @rajeevsathish

**Comment by Aparna-karri591** [comment]
Hello ,

Currently we are not able to provide the Sign Off since [CO-172](https://project-sunbird.atlassian.net/browse/CO-172) QA Required ticket is not yet released


cc: @vinukumar-vs @chitranshu-keshav @ShailandraOjha @Nadiya496 @ashokreddy1208 @mmanimegalai @arun1205 @vaivk369 @Krishnaj20 @rajeevsathish

**Comment by Aparna-karri591** [comment]
**R 5.1.3.0 Sunbird CoKreat - Development Phase Conditional Sign Off** 

Hello All,

This is Development Phase QA Conditional Sign off for the release 5.1.3.0  Sunbird CoKreat Building block.

Environment: Dock Staging
Testing Status: Completed
Defects Dashboard: NA

Execution Sheet:[Link](https://docs.google.com/spreadsheets/d/10xUWWmkWAMueVOzSUcQCiGxEKKulh3tQCWaNuqwuq5Q/edit#gid=592833374)

Scope:
Release 5.1.1.0 : [Link](https://project-sunbird.atlassian.net/issues/?filter=12693)
Release 5.1.2.0 : [Link](https://project-sunbird.atlassian.net/issues/?filter=12716)
Release 5.1.3.0 : [Link](https://project-sunbird.atlassian.net/issues/?filter=12736)

Building Block Sign Off Template: [Link](https://docs.google.com/spreadsheets/d/10xUWWmkWAMueVOzSUcQCiGxEKKulh3tQCWaNuqwuq5Q/edit#gid=1487068805)


**Note**: We are giving conditional Sign off since [CO-93](https://project-sunbird.atlassian.net/browse/CO-93) This Ticket has dependency on Inquiry BB

cc: @vinukumar-vs @chitranshu-keshav @ShailandraOjha @Nadiya496 @ashokreddy1208 @mmanimegalai @arun1205 @vaivk369 @Krishnaj20 @rajeevsathish

**Reply by Aparna-karri591** [reply]
@vinukumar-vs, @vaivk369 We are given the sign off for R5.1.3.0, Now  waiting for the deployment with Inquiry R5.3 and R5.4 Changes to start R5.2 Sunbird Cokreat regression

CC: @chitranshu-keshav @ShailandraOjha @Nadiya496 @ashokreddy1208 @mmanimegalai @gandham-santhosh
